core/src/main/java/org/hisp/dhis/android/core/program/ProgramCall.java

```diff
@@ -208,7 +209,8 @@ public boolean isExecuted() {
                                         DataElement.deleted, DataElement.dimension, DataElement.displayFormName,
                                         DataElement.domainType, DataElement.formName, DataElement.numberType,
                                         DataElement.valueType, DataElement.zeroIsSignificant,
-                                        DataElement.optionSet.with(OptionSet.uid, OptionSet.version)
+                                        DataElement.optionSet.with(OptionSet.uid, OptionSet.version),
+                                        DataElement.categoryCombo.with(CategoryCombo.uid)
```

Comment:
@dhis2-android you're right, we introduced it there because aggregate data was going to need it, and we suppose they're using the same model, so left it for you to merge it. We can remove it for next PR.